executor/point_get.go

```diff
@@ -247,6 +257,8 @@ func (e *PointGetExecutor) Next(ctx context.Context, req *chunk.Chunk) error {
 		}
 		return nil
 	}
+	e.memTracker.Consume(int64(cap(val)))
```

Comment:
`val` is just a local variable, and will be released after `Next()` returned.

I think we need tracking usage of executor's significant memory consuming members, such as  `idxKey`, `handleVal`.

Reply:
@zhaoxugang PTAL this comment. Rest LGTM.

Reply:
IMO the point_get_exec mabe should't track memory,because the memory tracker should track the data from store but the data from store is return to client immediately.and the idxKey, handleVal is only one for point_get, it's Constant level so should't track.

Reply:
Why we should not track memory at constant level ?

Reply:
You are right...I have done